packages/picom/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/yshui/picom
+TERMUX_PKG_DESCRIPTION="A lightweight compositor for X11"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yisus7u7 <jesuspixel5@gmail.com>"
+TERMUX_PKG_VERSION=8.2
+TERMUX_PKG_SRCURL=https://github.com/yshui/picom/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9d0c2533985e9670ff175e717a42b5bf1a2a00ccde5cac1e1009f5d6ee7912ec
+TERMUX_PKG_DEPENDS="libx11, libxcomposite, libxdamage, libxext, libxfixes, libxrender, xorgproto, xcb-proto, libconfig, libxdamage, libxcb, libxfixes, xcb-util-renderutil, libxrandr, libxcomposite, xcb-util-image, libxinerama, libpixman, dbus, pcre, libev, uthash"
+
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS(){
+	-Dopengl=false
+	}
```

Comment:
I'm strongly recommending to visit https://github.com/termux/termux-packages/wiki/Creating-new-package#table-of-available-package-control-fields and also learn shell scripting. The lines 10,11,12 contain statement that is... simply wrong.

Reply:
I have already checked and still do not understand the error. 
Isn't that function supposed to pass arguments to the build system?

Reply:
Do you understand the difference between variable and function?

Reply:
Even if it is a function, the command `-Dopengl=false` doesn't exist anyway. So what you have made is wrong.

However that field really is not supposed to be a function and can't be used as such. This is variable.